include/zenoh_commons.h

```diff
@@ -763,6 +763,9 @@ typedef struct z_moved_keyexpr_t {
  */
 #if defined(Z_FEATURE_UNSTABLE_API)
 typedef struct z_liveliness_subscriber_options_t {
+  /**
+   * Receive liveliness tokens that were declared before this liveliness subscriber.
```

Comment:
maybe "Whether to receive state change notifications for liveliness tokens that were declared before this liveliness subscriber.", or something like "If true, subscriber will receive the state change notifications for liveliness tokens that were declared before its declaration" ?